src/app/paging/paging-virtual.component.scss

```diff
@@ -4,3 +4,7 @@
   height: 50px;
   text-align: center;
 }
+
+::ng-deep .datatable-footer .datatable-pager a {
```

Comment:
Don't use `ng-deep`. you can simply override the css variable instead of overriding specific style